apps/stable_diffusion/web/ui/txt2img_ui.py

```diff
@@ -193,6 +193,7 @@ def txt2img_inf(
             dtype,
             args.use_base_vae,
             cpu_scheduling,
+            args.max_embeddings_multiples,
```

Comment:
This should happen earlier when the Text2ImagePipeline is initialized. 
As is the seqlen64 unet/CLIP is selected and the prompt is truncated before `max_embeddings_multiples` is set to 5 when you run via the webui.

Reply:
Are you sure? I've verified it's using unet512 from the webui and it's producing identical images.

Reply:
Yeah thanks for taking a look. May have just needed clearing cache etc.

Good to go now.